news/_posts/releases/2018-06-13-prestashop-1-7-5-0-beta-release.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: post
+title:  "PrestaShop 1.7.5.0 beta is open for feedback!"
+subtitle: "You can download it and test it"
+date:   2018-11-06 17:45:00
+authors: [ PrestaShop ]
+icon: icon-leaf
+tags:
+ - version
+ - beta
+ - patch
+ - releases
+ - 1.7.5
+---
+
+PrestaShop 1.7.5 beta is available for download. Please read important informations about it.
+
+## New & improvements
+
+### Features
+
+- SEO improvements in category page :
+  - Change in pagination
+  - Category blocked not displayed after page 1
+  - SEO improvement in brands & suppliers urls : _name is replaced by /brand/x-name and /supplier/x-name 
+- Product page :
+  - Specific prices can now be edited
+  - The SEO section now displays a preview so that you can see how your page would be shown in Google results
+  - When a product is taken offline, by default visitors will be redirected to the product’s main category, instead of a “Not found” page.
+  - Stock location field now available
+- New module manager page with new categorization (Addons categories) and ability to manage modules by last use
+
+
+### UX
+
+- UX improvement on Module pages
+- Helper card in SEO & URL page
+
+### New & fixed modules
+
+- The gsitemap module is now available for PrestaShop 1.7.
+- Favicon notifications in Back Office (native module)
+- Buy Button Lite (native module)
+- Cross selling module (native)
+- Viewed products (native)
+
+
+### Installer
+
+- During install, if there is a more recent stable version of PrestaShop available, it is possible to download and install the latest instead
+- New languages in installer:
+  - Hindi
+  - Bosnian
+  - Spanish Mexico
+ 
+
+## The technical side
+
+### Symfony migration
+
+New pages:
+- Orders -> Delivery slips
+- Orders -> Invoices
+- Design -> Theme Catalog
+- Design -> Positions
+- Shipping -> Preferences
+- Payment -> Payment Methods
+- Payment -> Preferences
+- International -> Localization -> Localization
+- International -> Localization -> Geolocation
+- International -> Translations (Page ½)
+- Shop Parameters -> Order Settings -> Order Settings
+- Shop parameters > Traffic & SEO > SEO & URLs
+- Advanced Parameters -> Database -> SQL Manager
```

Comment:
I suggest we do not talk about "Advanced Parameters -> Database -> SQL Manager" pages as they have been hidden in 1.7.6 because of a complex issue we cannot solve in time for the release.